terraform/ec2/amis.tf

```diff
@@ -380,6 +379,10 @@ EOF
   }
 }
 
+# Local variables only apply to aws_ami for the customized filter input
+locals {
+  arch = var.amis[var.testing_ami]["arch"] == "amd64" ? "x86_64" : var.amis[var.testing_ami]["arch"]
```

Comment:
Do we still need 'x86_64' here in this case

Reply:
yes, the terraform ami data filter only use x86_64 as parameter instead of amd64